TrajectoryPlannerROS:

# Robot Configuration Parameters
  max_vel_x: 0.15
  min_vel_x: 0.08

  max_vel_y: 0.15
  min_vel_y: 0.08

  max_vel_theta:  0.5
  min_vel_theta: -0.5
  min_in_place_vel_theta: 0.1

  acc_lim_x: 1.0
  acc_lim_y: 1.0
  acc_lim_theta: 1.5

# Goal Tolerance Parameters
  xy_goal_tolerance: 0.05
  yaw_goal_tolerance: 0.05

# omni robot configuration
  holonomic_robot: false

# Forward Simulation Parameters
  sim_time: 0.8
  vx_samples: 18
  vtheta_samples: 20
  sim_granularity: 0.05
